target/linux/mediatek/dts/mt7986a-acer-predator-w6d.dts

```diff
@@ -0,0 +1,146 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+/dts-v1/;
+#include "mt7986a-acer-w6-common.dtsi"
+
+/ {
+	model = "Acer Predator Connect W6d";
+	compatible = "acer,predator-w6d", "mediatek,mt7986a";
+
+	aliases {
+		serial0 = &uart0;
+		led-boot = &led_status_blue;
+		led-failsafe = &led_status_blue;
+		led-running = &led_status_green;
+		led-upgrade = &led_status_red;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		led_status_red: led0 {
+			label = "ant0:red";
```

Comment:
Please convert `label` properties of LEDs to `color` and `function`.

```
color = <LED_COLOR_ID_*>;
function = LED_FUNCTION_*;
function-enumerator = <N>;
```

or 

```
color = <LED_COLOR_ID_*>;
function = "function";
function-enumerator = <N>;
```

And, node names of LEDs should be `led-<N>` format.

https://elixir.bootlin.com/linux/v6.6.59/source/Documentation/devicetree/bindings/leds/common.yaml

BTW, `"<color>:<func>"` format should be used for `label` property.